### ISSUE TITLE
Release 12.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5457]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5457|
|Type|task|
|Created|2012-01-22 21:20:27Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-12.0.0-5457|

I know, already?!

<details><summary>Searchable metadata</summary>

```
trac-id__5457 5457
type__task task
reporter__therve therve
priority__normal normal
milestone__Twisted_12_0 Twisted-12.0
branch__releases_release_12_0_0_5457 releases/release-12.0.0-5457
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1327267227000000 1327267227000000
changetime__1328968069000000 1328968069000000
version__None None
owner__therve therve
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve removed owner|
|-|-|

Please review!

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_4"></a>|@jml set owner to @therve|
|-|-|

Thanks for doing this.

It took a while for me to figure out how to review again, on account of not having Combinator on this machine and not using Subversion in a _long_ time. Overall it was about half an hour from starting to having a diff that I felt comfortable reviewing.

Anyway,

 * The change in setup.py implies that this release is the first to not support Python 2.4. Probably worth calling that out in the release email.

 * `test_assertions.py` still has a 2011 copyright header.  I don't think this needs to be fixed in this branch, but IANAL.

So, no necessary changes to this branch for it to land.

Very glad we're getting such an early release for 2012.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [33539]) Merge release-12.0.0-5457

Author: therve
Reviewer: jml
Fixes: [#5457](https://github.com/twisted/twisted/issues/5457)

Release Twisted 12.0.0.
